crates/evm/src/system_calls/eip7002.rs

```diff
@@ -118,6 +118,11 @@ where
     // commit the state
     evm.context.evm.db.commit(state);
 
+    post_commit(result)
+}
+
+#[inline]
+pub(crate) fn post_commit(result: ExecutionResult) -> Result<Vec<Request>, BlockExecutionError> {
```

Comment:
can we document here that this parses the requests from the execution output?

Reply:
sure, done here https://github.com/paradigmxyz/reth/pull/11068/commits/7cad4380d5af1112b954126d146b81161cb56f56